src/SFA.DAS.Reservations.Api.AcceptanceTests/Steps/GetAccountLegalEntitiesSteps.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using SFA.DAS.Common.Domain.Types;
+using SFA.DAS.Reservations.Api.Controllers;
+using SFA.DAS.Reservations.Data;
+using SFA.DAS.Reservations.Domain.Entities;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.Reservations.Api.AcceptanceTests.Steps
+{
+    [Binding]
+    public class GetAccountLegalEntitiesSteps : StepsBase
+    {
+        private IEnumerable<Domain.AccountLegalEntities.AccountLegalEntity> _accountLegalEntitiesList;
```

Comment:
please don't do this. get and set the account legal entities via testdata

Reply:
Because this is a result from one of the steps it was originally only used within the scope of the steps class by design. I've moved it over and kept that scope for now as its only used by these steps and any other steps can still get the results from just re-quering the database. We can put it in the test data class but I'm wondering if we should be storing test setup data and results data separately if we wanted to make them globally accessible? just in case one steps overwrites the test setup data of a previous step.

Reply:
I'd go so far as to say 
`Given the following account legal entities`
`##a list of ales here, one of them without agreement, the other with`
`When calling get api/{id_used_in_previous_step}/accountlegalentities`
`Then account legal entities returned are`
`##a list of ales here, one of them without agreement, the other with`

The given is setting up the data context 
The when can store the result into a testdata result property
The then will read the result property and assert that each specified ale is present.

Reply:
We could do that, for now i'll keep it simple and add a test result model (as discussed in Slack) and then we can have a team chat about how we want to deal with results in general

Reply:
We can also chat about adding those accounts via the scenarios as well